peripherals/wwdg/wwdg.yaml

```diff
@@ -0,0 +1,38 @@
+# The most common features, found on Basic timers TIM6 and TIM7,
+# and found on all more advanced timers.
+
+WWDG:
+  CR:
+    _modify:
+      WDGA: 
+        description: "Activation bit for the watchdog. This bit is set by software and only cleared after a reset."
+        bitOffset: 7
+        bitWidth: 1
+      T:
+        description: "These bits contain the value of the watchdog counter. A reset is produced when it rolls over from 0x40 to 0x3F (T6 us cleared)."
+        bitOffset: 0
+        bitWidth: 7
+  CFR:
+    WDGTB:
+      Div1: [0, "Counter clock (PCLK1 div 4096) div 1"]
+      Div2: [1, "Counter clock (PCLK1 div 4096) div 2"]
+      Div4: [2, "Counter clock (PCLK1 div 4096) div 4"]
+      Div8: [3, "Counter clock (PCLK1 div 4096) div 8"]
+      _merge:
+        - "WDGTB*"
+    _modify:
+      EWI:
+        description: "Used to enable early wakeup interrupt. This allows an interrupt to be serviced before device reset. Only cleared after a reset"
+        bitOffset: 9
+        bitWidth: 1
+      W:
+        description: "Window value to be compared to the downcounter."
+        bitOffset: 0
+        bitWidth: 7
+  SR:
+    EWIF:
```

Comment:
Annoyingly this field is called `EWI` on some of the devices you've added the WWDG peripheral to (I think in all the STM32F1 family), which is why the automatic build is failing. The best thing is probably to add a fix to the F1 device files, like:
```
WWDG:
  SR:
    _modify:
      EWI:
        name: EWIF
```
You could add a note that the field in the SVD is named incorrectly compared to the reference manual (RM0008).